### PR TITLE
Good bye! image and remote_image banner, welcome media

### PR DESCRIPTION
**Pull request by tarunbhardwaj**
Added provision to use different type of media like video, audio as
banner not just image.

### CODE REVIEW

File: cms.py [hunk 1]
```diff
@@ -313,24 +314,29 @@ class Banner(Workflow, ModelSQL, ModelView):
 
     # Type related data
     type = fields.Selection([
-        ('image', 'Image'),
-        ('remote_image', 'Remote Image'),
+        ('image', 'Image'),  # TODO: Deprecate in 3.6
+        ('remote_image', 'Remote Image'),  # TODO: Deprecate in 3.6
+        ('media', 'Media'),
+        ('remote_media', 'Remote Media'),
```
Comment (prakashpp) [inline cms.py]: I dont think this is needed. Like nereid static files we should remove this, remote images cant be transformed.

File: cms.py [hunk 2]
```diff
@@ -408,13 +413,17 @@ def get_html(self):
 
         banner = self.read(
```
Comment (prakashpp) [inline cms.py]: why read? `self` is active record

File: cms.py [hunk 3]
```diff
@@ -408,13 +413,17 @@ def get_html(self):
...
-                'type', 'click_url', 'file',
-                'remote_image_url', 'custom_code', 'height', 'width',
+                'type', 'click_url', 'file', 'file.mimetype',
+                'custom_code', 'height', 'width',
                 'alternative_text', 'click_url'
             ]
         )[0]
 
-        if banner['type'] == 'image':
+        if banner['type'] in ['image', 'media']:
```
Comment (prakashpp) [inline cms.py]: image?

File: cms.py (hunk 2 above)
Comment (tarunbhardwaj) [inline cms.py]: for performance.

Comment (prakashpp) [inline cms.py]: Could you please write a comment over it?

**Pull request merged**